src/util/StringUtils.h

```diff
@@ -381,6 +385,33 @@ vector<string> split(const string& orig, const char sep) {
   return result;
 }
 
+// _____________________________________________________________________________
+vector<string> splitWs(const string &orig) {
+  vector<string> result;
+  if (orig.size() > 0) {
+    size_t start = 0;
+    size_t pos = 0;
+    while (pos < orig.size()) {
+      if (orig[pos] >= 0 && ::isspace(orig[pos])) {
+        if (start != pos) {
+          result.emplace_back(orig.substr(start, pos - start));
+        }
+        // skip any whitespace
+        while (pos < orig.size() && orig[pos] >= 0 && ::isspace(orig[pos])) {
+          pos++;
+        }
+        start = pos;
+      }
+      pos++;
+    }
+    // avoid adding whitespace at the back of the string
+    if (!(orig[orig.size() - 1] >= 0 && ::isspace(orig[orig.size() - 1]))) {
```

Comment:
this can be replaced by just `if (start != orig.size())` because we would already have skipped the whitespace. Then it also more closely matches the condition `if (start != pos)` in the loop which basically does the same thing.